services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.js

```diff
@@ -126,8 +126,11 @@ const Library = ({ libraryData, params }) => {
                       />
                     </dd>
                   </Column>
-                  <Button className={styles['versions-button']}>
-                    Coming soon...
+                  <Button
+                    href={`/assets/${libraryData.params.library}/library-assets`}
```

Comment:
We probably want to use either a next link here or a router.push to avoid the full page reload